sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/models/TaggedBlobItem.java

```diff
@@ -15,8 +18,19 @@
      * @param name The blob name
      */
     public TaggedBlobItem(String containerName, String name) {
+        this(containerName, name, new HashMap<>());
```

Comment:
Should we use `Collections.emptyMap()` here to use a singleton instead of creating a new instance?